python/ray/serve/tests/test_pipeline_dag.py

```diff
@@ -151,17 +142,17 @@ def func_2(input):
     with pytest.raises(ValueError, match="Please provide a driver class"):
         _ = serve.run(serve_dag)
 
-    handle = serve.run(Driver.bind(serve_dag))
-    assert ray.get(handle.remote(2)) == 6  # 2 + 2*2
+    handle = serve.run(DAGDriver.bind(serve_dag))
+    assert ray.get(handle.predict.remote(2)) == 6  # 2 + 2*2
```

Comment:
for each test case that we're calling the python app handle, can we also make a http call ?